pages/search/search.jsx

```diff
@@ -19,6 +19,9 @@ export default React.createClass({
   },
   componentDidMount() {
     this.setState({ keywordSearched: this.props.location.query.keyword });
+    // to select the <input /> rendered from <DebounceInput /> we have to select it by its id.
```

Comment:
no we don't. We can put a `ref="search"` on the `<DebounceInput>` and then we can call whatever function it needs called to focus directly by using `this.ref.search.focus()` (assuming that's the function it exposes)